app/models/assignment.rb

```diff
@@ -24,6 +24,8 @@ class Assignment < Assessment
   before_save :reset_collection_time
   after_create :update_parent_assignment, if: :is_peer_review?
 
+  delegate :enable_student_tests, :token_period, to: :assignment_properties
```

Comment:
This line isn't necessary, as the `delegate_missing_to` above will handle these two attributes